OpenRA.Mods.Common/Activities/Air/ReturnToBase.cs

```diff
@@ -120,7 +120,7 @@ public override bool Tick(Actor self)
 
 				aircraft.MakeReservation(dest);
 				QueueChild(new Land(self, Target.FromActor(dest), offset, facing, Color.Green));
-				QueueChild(new Resupply(self, dest, WDist.Zero, alwaysLand));
```

Comment:
I don't have time to review this in-depth, but someone should test whether this really doesn't regress things with aircraft, since I don't see anything aircraft-specific that restores `WDist.Zero` and I faintly remember we used that over 512 for a reason.
Maybe overriding the default at yaml level would be enough, but in any case this is something that needs to be looked into before merging this.

Reply:
I have looked at it and this doesn't matter for now that we land at the centre of the structure. RTB activity manages aircraft landing and not Resupply, so it will only become an issue when we merge those 2 activities together

Reply:
If I read https://github.com/OpenRA/OpenRA/pull/15543/files#diff-6cc1af605125ba76cf4f05e5ce3e41d3fc1f73bc7b9f5729737b332e937b1fb0R56 right, 512 is actually the exception while `WDist.Zero` was the intended/default case that non-aircraft deviate from.

Reply:
WDist.Zero is for aircraft because they can land in a way that WDIst from the centre is 0, but ground units often can't do that. But there's no harm in having it match for both

Reply:
keep in mind that all this code (RTB and Resupply) will be removed, `DockSequence` will be used instead

Reply:
The thing is why can't ground units do that? (This is a very old problem: see the HACK/TODO comments.) The harm that could happen is aircraft not going to the center, although I don't see an indication of that happening right now.

Reply:
I've actually explained this before. Resupply goes to the centre of the actor, not the the centre cell. It's not ideal as then we can't have custom offsets. We will be moving to a different cell based system that is currently used by Refinery later on

Reply:
Yes, which means the old HACK/TODO is still in place, doesn't it?

Reply:
Sure, I'll add in a comment where 512 is defined